apis/fluentbit/v1alpha2/plugins/parser/regex_types.go

```diff
@@ -39,8 +41,11 @@ func (re *Regex) Params(_ plugins.SecretLoader) (*params.KVs, error) {
 	if re.TimeKeep != nil {
 		kvs.Insert("Time_Keep", fmt.Sprint(*re.TimeKeep))
 	}
+	if re.TimeKeep != nil {
```

Comment:
I think you mean `TimeOffset` here, not `TimeKeep`.
`TimeKeep` exists in the lines above

Reply:
It's fixed in the second commit

Reply:
Got it